main.tf

```diff
@@ -0,0 +1,28 @@
+#####################################################
+# IBM Cloud Resource Management
+# Copyright 2021 IBM
+#####################################################
+
+###################################################################
+# Create resource group
+###################################################################
+module "resource-group" {
```

Comment:
Why we have module block for resource-group and resource for tagging in main.tf

Reply:
Removed module block and added resource.